td4m3/src/main/res/layout/accounts_view.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Where are you calling this?

Reply:
No where. I'm not sure where to call it. Any ideas?